lib/client.ts

```diff
@@ -20,19 +21,17 @@ export function client (config: {
     invalid('client was initiated without a config object')
   );
 
-  const {
-    token,
-    baseURL = 'https://api.truework.com/',
-    version = '2019-10-15',
```

Comment:
This is _technically_ a breaking change, since we were not actually using the latest API version when a version param is not passed, and now we will. However, this is aligned with what the documentation says, so not sure if we want to version it as a breaking change or not.

Reply:
Good call, but yeah I think we're OK there.